app/src/main/java/org/simple/clinic/selectcountry/SelectCountryModel.kt

```diff
@@ -1,3 +1,16 @@
 package org.simple.clinic.selectcountry
 
-object SelectCountryModel
+data class SelectCountryModel(
+    val supportedCountries: List<Country>?
```

Comment:
Use `List<Country>` with an `emptyList()` instead of a nullable property?

Reply:
This was intentional. A nullable list indicates that the manifest has not yet been fetched. An empty list indicates that the manifest has been fetched, but there were no countries in the manifest.

Does this make sense?

Reply:
This might also change when we implement the `Update` since we will need to display network errors, so we could change it then.